public/markdown/community-process.md

```diff
@@ -0,0 +1,95 @@
+
+For this reason, Typesafe's relationship to the Play project is best described as a stewardship, Typesafe manages the Play project, but is held accountable by the Play community.
+
+For a statement of how Typesafe views open source, see the [Typesafe Open Source Position Statement](http://typesafe.com/open-source-position-statement).
+
+## Definitions
+
+### Contributors
+
+A contributor is anyone that makes a contribution to Play.  This does not necessarily mean code contributions, it could mean any of the following:
+
+* Code fixes, improvements and new features
+* Documentation fixes, improvements and new features
+* Translating documentation into other languages
+* Writing or maintaining plugins, modules or libraries for Play
+* Code reviews in GitHub
+* Raising, triaging and adding additional information to help resolve issues
+* Taking part in design and feature discussions on mailing lists
+* Answering questions on mailing lists and in stack overflow
+* Running, speaking at or contributing to user groups focussed on Play Framework
+* Speaking at conferences, blogging about or otherwise promoting Play Framework
+
+### Integrators
+
+An intergrator is anyone with write access to the source code and documentation of the Play project, or one of the projects that come under the playframework GitHub organization.
+
+It should be noted that you do not have to be an integrator to contribute to Play, and in fact there is nothing in the list of things considered to be contributions that you can't do if you are not a integrator.  In practice, the only thing that integrators can do that contributors can't are adminstration type tasks, such as merging contributions from other contributors, and house keeping tasks in the issue tracker such as closing fixed or invalid issues.
+
+## Decision making
+
+Decisions in the Play project fall in two broad categories:
+
+* Implementation decisions, this includes whether a pull request is up to standard to be merged, how a feature or improvement should be implemented.
+* Design and house keeping decisions, aka everything else.  This includes major design decisions, road maps, release schedules, decisions about how the project should be run and managed, what tools should be used, etc.
+
+### Design and house keeping decisions
+
+The primary place for discussion about the design of Play and how the Play project is run is the [play-framework-dev](https://groups.google.com/forum/#!forum/play-framework-dev) mailing list.  All major new features, refactorings or changes to the project should first be discussed in this forum.  The aim of the discussions is to reach an understanding on whether the task will be done, and how it will be done.  When a new topic is posted, interested parties are encouraged to comment with their affirmation or concerns.
```

Comment:
I don't do a very good job of using the dev mailing list ot communicate what I'm working on. I'll try and do more talking there.